cmd/api.go

```diff
@@ -78,6 +81,11 @@ func (o *apiOptions) runAddAPI() {
 		o.apiScaffolder.DoController = util.Yesno(reader)
 	}
 
+	if !o.controllerFlag.Changed {
+		fmt.Println("Create Webhook [y/n]")
```

Comment:
Which webhooks does this create?  defaulting?  validating?  conversion?  we should spell this out in the message (especially since more "manual" webhooks might be different)